pkg/nsx/auth/jwt/jwtcache.go

```diff
@@ -47,6 +47,7 @@ func (cache *JWTCache) GetJWT(refreshToken bool) (string, error) {
 	jwt, err := cache.refreshJWT()
 	if err != nil || jwt == "" {
 		log.V(1).Info("get JWT", "Refresh JWT error", err)
+		cache.tesClient.signer = nil
 		return "", err
 	}
 	exp, err := cache.getJWTExpire(jwt)
```

Comment:
add log and set cache.expire = exp if err != nil
so next time getJWT is called it will find the token has been expired.
reset cache.tesClient.signer just in case that need to reload credentials and create new VAPI session.